bumpversion/__init__.py

```diff
@@ -12,24 +12,32 @@
 except:
     from io import StringIO
 
-
 import argparse
+import codecs
+import io
 import os
+import platform
 import re
 import sre_constants
 import subprocess
+import sys
 import warnings
-import io
 from string import Formatter
 from datetime import datetime
 from difflib import unified_diff
 from tempfile import NamedTemporaryFile
 
-import sys
-import codecs
-
 from bumpversion.version_part import VersionPart, NumericVersionPartConfiguration, ConfiguredVersionPartConfiguration
 
+
+if platform.system() == 'Windows' and sys.version_info[0] == 2:
+    def _command_args(args):
```

Comment:
Would it make sense to introduce a `subprocess.check_output` wrapper that does this instead?

Reply:
That might be more readable, I was just going for as precise a change as needed.